test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightParameterReaderTests.cs

```diff
@@ -322,6 +323,124 @@ public void ReadFeed()
         }
 
         [Fact]
+        public void ReadTwoFeeds()

```

Comment:
2 feeds with different types would be an important scenario for Dynamics CRM as well.